File: 0706-design-hashmap/0706-design-hashmap.cpp
class MyHashMap {
public:
    int freq[1000001]; //  1000001;
    MyHashMap() {
        fill (freq , freq + 1000001 , -1);
    }
    
    void put(int key, int value) {
        freq[key] = value;
    }
    
    int get(int key) {
        return freq[key];
    }
    
    void remove(int key) {
        freq[key] = -1;
    }
};

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap* obj = new MyHashMap();
 * obj->put(key,value);
 * int param_2 = obj->get(key);
 * obj->remove(key);
 */

File: 0706-design-hashmap/0706-design-hashmap.cs
public class MyHashMap {
    int [] freq ;
    public MyHashMap() {
        freq = new int [1000001];
        Array.Fill (freq , -1);
    }
    
    public void Put(int key, int value) {
        freq[key] = value;
    }
    
    public int Get(int key) {
        return freq[key];
    }
    
    public void Remove(int key) {
        freq[key] = -1;   
    }
}

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap obj = new MyHashMap();
 * obj.Put(key,value);
 * int param_2 = obj.Get(key);
 * obj.Remove(key);
 */